src/components/Map.js

```diff
@@ -85,12 +86,11 @@ class Map extends Component {
 
       // For each marker created, add a listener that checks for clicks
       google.maps.event.addListener(marker, 'click', function (){ //eslint-disable-line
-        if (currentSelectedMarker) {
-          currentSelectedMarker.message.close();
-        } else {
-          context.props.changeCurrEvent(pin.eventIdx);
-        }
-        currentSelectedMarker = pin;
+        if (context.currentSelectedMarker) {
+          context.currentSelectedMarker.message.close();
```

Comment:
I cannot locate the function currentSelectedMarker.message.close(). Could you point me to where this function and where .message is assigned to?